### PR TITLE
[IMP] pos : add helpers for test

### PR DESCRIPTION
**Pull request by SalukiMakingCode**
Add helpers for test to access menu options

tasks id : 3674898

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_restaurant/static/tests/tours/FloorScreen.tour.js [hunk 1]
```diff
@@ -26,11 +26,7 @@ registry.category("web_tour.tours").add("FloorScreenTour", {
 
             // clicking table in active mode does not open product screen
             // instead, table is selected
-            Chrome.clickMenuButton(),
-            {
-                content: `click edit button`,
-                trigger: `.toggle-edit-button`,
```
Comment (vlst-odoo) [inline addons/pos_restaurant/static/tests/tours/FloorScreen.tour.js:32]: i think you can now remove the `toggle-edit-button` class from the xml too

File: addons/point_of_sale/static/tests/tours/helpers/ChromeTourMethods.js [hunk 2]
```diff
@@ -21,6 +21,15 @@ export function clickMenuButton() {
         trigger: ".menu-button",
     };
 }
+export function clickMenuOption(name) {
```
[the remark this answers is not in the document]
Comment (SalukiMakingCode) [reply addons/point_of_sale/static/tests/tours/helpers/ChromeTourMethods.js:24]: I change the function, but it's strange : I try to test it, but it seems that this function is never really use

[the remark this answers is not in the document]
Comment (vlst-odoo) [reply addons/point_of_sale/static/tests/tours/helpers/ChromeTourMethods.js:24]: that function might indeed not be used. If that is the case we can delete it. There are still many places where we can use your new function though 
image

[the remark this answers is not in the document]
Comment (SalukiMakingCode) [reply addons/point_of_sale/static/tests/tours/helpers/ChromeTourMethods.js:24]: Two of those looks like it's the same check, but is not ;)

**Review by vlst-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**